devicon.json

```diff
@@ -1082,6 +1082,22 @@
             }
         ]
     },
+    {
+        "name": "figma",
+        "tags": [
+            "design"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#f24e1e",
```

Comment:
There are no plain versions for this icon. Please add a font version for the icon (see the [Jupyter](https://github.com/devicons/devicon/tree/develop/icons/jupyter) icon for example).